diff --color -ruN a/include/fmt/core.h b/include/fmt/core.h
--- a/include/fmt/core.h	2024-01-03 14:17:28.000000000 -0500
+++ b/include/fmt/core.h	2024-07-03 19:45:02.454527551 -0400
@@ -546,7 +546,7 @@
 }
 template <typename Char,
           FMT_ENABLE_IF(!std::is_empty<std_string_view<Char>>::value)>
-inline auto to_string_view(std_string_view<Char> s) -> basic_string_view<Char> {
+constexpr auto to_string_view(std_string_view<Char> s) -> basic_string_view<Char> {
   return s;
 }
 template <typename S, FMT_ENABLE_IF(is_compile_string<S>::value)>
